Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/API/wmb.yaml

```diff
@@ -0,0 +1,90 @@
+imports:
+  Text: Data.Text
+  APISuccess: Kernel.Types.APISuccess
+  LatLong: Kernel.External.Maps.Types
+
+module: WMB
+
+types:
+  RouteInfo:
+    code: Text
+    shortName: Text
+    longName: Text
+    startPoint: LatLong
+    endPoint: LatLong
+
+  AvailableRoutesList:
+    routeCode: Text
+    source: RouteInfo
+    destination: RouteInfo
+
+  StopInfo:
+    name: Text
+    code: Text
+    lat: Maybe Double
+    long: Maybe Double
+
+  BusType:
+    enum: "AC,NON_AC"
+
+  BusDetails:
+    number: Text
+    tyep: BusType
+
+  TripLinkReq:
+    vehicleNumber: Text
+    routeCode: Text
+
+  TripLinkResp:
+    tripTransactionId: Text
+    busDetails: BusDetails
```

Comment:
Instead of Bus we can mention Vehicle to make it generic for Bus/Metro/Suburban etc.